src/profile/components/multiItemRow/multiItemRow.module.css

```diff
@@ -0,0 +1,43 @@
+.rowContentWrapper {
+  background: var(--color-black-ultra-light);
+  margin: 0 0 var(--spacing-m) 0;
+  padding: var(--spacing-m);
+}
+
+.rowContentWrapper form {
+  width: 100%;
+}
+
+.editableRow {
```

Comment:
Airbnb styleguide prefers dashes over camelCasing in class names: https://github.com/airbnb/css#css

dev.hel.fi recommends following the airbnb's style guide with some exceptions: https://dev.hel.fi/coding-standards#reactjsx-style-guide

Doesn't refer to css there though, so not sure if it should be followed in styles also?

Reply:
I have kept the same naming that all other css files follow in the Profile UI. Original files are probably older than the spec in dev.hel.fi. Maybe all files should be changed in one update or just this one now?

Maybe combine this to a HDS update.

Reply:
Yeah it might be a big task to change them all, so not sure is it worth it. Though if we're still in the beginning part of creating the open-city-profile-ui, it might be worth it.

Reply:
There wont be any new css files :)